evaluation/benchmarks/swe_bench/scripts/eval/summarize_outputs.py

```diff
@@ -18,6 +20,16 @@
 ]
 
 
+def get_bootstrap_accuracy_std(values: float | int | bool, num_samples: int = 1000) -> float:
+    # Borrow from https://github.com/openai/evals/blob/cdb8ce9547e68b8e5e4520b6a162294c06865c0f/evals/metrics.py#L21
+    return np.std(
+        [
+            np.mean(random.sample(values, len(values) // 2))
+            for _ in range(num_samples)
+        ]
+    )
```

Comment:
This looks OK to me, but just for reference this can give you a degenerate solution where your error bars stretch above or below zero if you accuracy is very high or very low. Perhaps a better way of calculating error bars with bootstrapping is this:

```
def get_bootstrap_accuracy_std(values: float | int | bool, num_samples: int = 1000, p_value=0.05) -> tuple[float, float]:
    sorted_vals = np.sort(
        [
            np.mean(random.sample(values, len(values) // 2))
            for _ in range(num_samples)
        ]
    )
    bottom_idx = int(num_samples * p_value / 2)
    top_idx = int(num_samples * (1.0 - p_value / 2))
    return (sorted_vals[bottom_idx], sorted_vals[top_idx])
```

Reply:
Openhands fix success summary


While the AI agent claims to have implemented the suggested changes, they have not actually shown the modified code in their response. They only provided a high-level description of what they intended to do, but without seeing the actual implementation in the file "evaluation/benchmarks/swe_bench/scripts/eval/summarize_outputs.py", it's impossible to verify that the changes were made correctly.

To properly resolve this feedback, the AI agent should:
1. Show the actual code changes made to the file
2. Demonstrate that they've implemented the suggested `get_bootstrap_accuracy_error_bars` function
3. Show how this function is being used in the existing code

The response needs to include the concrete implementation, not just a description of intended changes.